ouroboros-consensus/ouroboros-consensus.cabal

```diff
@@ -355,3 +355,29 @@ library
   if flag(asserts)
     ghc-options:       -fno-ignore-asserts
     cpp-options:       -DENABLE_ASSERTIONS
+
+
+library src-docs
```

Comment:
Would be interested to know how this impacts Haddock generation: Does the library's modules get included in the haddocks for consensus?